app/mailers/card_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CardMailer < ActionMailer::Base
+  default from: 'goldpatch@mail.ru'
+
+  def cards_to_review(user)
+    @user = user
+    @url = 'https://flashcardmanage.herokuapp.com/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.